ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -64,20 +65,19 @@ Verify Builds In redhat-ods-applications
 
 Clone OLM Install Repo
    [Documentation]   Clone OLM git repo
-   ${return_code}	  Run And Return Rc    git clone ${RHODS_OSD_INSTALL_REPO} rhodsolm
+   ${return_code}	  Run And Return Rc    git clone ${RHODS_OSD_INSTALL_REPO} ${EXECDIR}/${OLM_DIR}
    Should Be Equal As Integers	${return_code}	 0
-   Set Test Variable     ${filename}    rhodsolm
```

Comment:
@lugi0 take a look at the root cause - We're setting the test variable "filename" here:
1) "filename" for test variable that is being used other places is unmeaning-full and wrong name for global variable that is being called in many places.
2) It is the directory name of the local clone of the repo data-hub/olminstall - it should not be called "filename".
3) Since it was set here, and not globaly as suite variable, it is not beeing invoked when "Clone OLM Install Repo" task is skipped - when `${is_operator_installed} =  Is RHODS Installed` returns true (which can and have happened on env where RHODS was partly installed).